addons/markers/functions/fnc_getEnabledChannels.sqf

```diff
@@ -40,12 +40,8 @@ if (_localize) then {
     if (setCurrentChannel 4) then {
         _enabledChannels pushBack localize "str_channel_vehicle";
     };
-
-    if (setCurrentChannel 5) then {
-        _enabledChannels pushBack localize "str_channel_direct";
-    };
 } else {
-    for "_i" from 0 to 5 do {
+    for "_i" from 0 to 4 do {
         if (setCurrentChannel _i) then {
             _enabledChannels pushBack _i;
         };
```

Comment:
If I understand correctly, this means that using this getter while in direct channel, the channel gets changed to global (or some other one), which would violate the no side effects principle of a getter.